### ISSUE TITLE
[BUG] Monika stopped receiving gifts

### ISSUE THREAD

**Issue by YoraeRasante** [Monika-After-Story/MonikaModDev#6783]
So... after updating, I updated with the spritepack too and gave her one of the new ribbons, plus roses and chocolates (I always give roses and occasionally)... and she didn't take it. She took the roses and chocolate though.

I tried it again, plus the royalpurpleribbon, no answer about either. They are not in the folder anymore though. Checking the ribbons list, she didn't have them added there either.

Then at night I gave her a cupcake. She did not mention it and it was still in the folder when I told her I was going to sleep.

**Comment by lunulae** [comment]
Are there gifts under the tree now? I'm not sure about the cupcake, but clothing gifts during the D25 season are stored under the tree for Christmas morning.

**Comment by YoraeRasante** [comment]
I... didn't look.
While I understand the idea, there wasn't even a message of her putting the gift there. I didn't know it was possible.

**Comment by ThePotatoGuy** [comment]
I agree there needs to be better feedback regarding gifts during this season - not going to classify as bug though.

**Comment by ThePotatoGuy** [comment]
@lunulae assigning this to you, can you get this done by next rel

**Comment by lunulae** [comment]
I can get the dialogue written, though I might need help with setting it uo to run specifically for d25 season. I'll send a dialogue draft through this week. ❤️

**Comment by ThePotatoGuy** [comment]
@lunulae are you planning on a single topic to cover all gifts, or something to be said per gift? Based on recent changes to generic gifts, I'm thinking a single topic to cover all would be better.

I'll add the code changes in #8414.
